src/main/resources/application.properties

```diff
@@ -135,6 +135,9 @@ search.elastic.data.hosts=${ELASTIC_SEARCH_DATA_NODES_HOSTS:"http://localhost:92
 search.blacklist=${ELASTIC_SEARCH_BLACKLIST:query_string}
 search.cases.index.name.format=${ELASTIC_SEARCH_CASE_INDEX_NAME_FORMAT:%s_cases}
 search.cases.index.name.type=${ELASTIC_SEARCH_CASE_INDEX_TYPE:_doc}
+search.elastic.nodes.discovery.enabled=${ELASTIC_SEARCH_NODES_DISCOVERY_ENABLED:true}
```

Comment:
do we need this enabled by default for local development?

Reply:
right, setting to false for local